add-on/chrome/test/mochitest/browser_panel_privateBrowsing.js

```diff
@@ -0,0 +1,20 @@
+/* Any copyright is dedicated to the Public Domain.
+   http://creativecommons.org/publicdomain/zero/1.0/ */
+
+"use strict";
+
+add_task(function* test_panel_should_be_shown() {
+  yield LoopUI.openPanel();
+
+  Assert.notEqual(document.getElementById("loop-notification-panel"), null, "Panel exists");
+});
+
+add_task(function* test_init_copy_panel_private() {
+  let win = OpenBrowserWindow({ private: true });
+  yield new Promise(resolve => win.addEventListener("load", resolve));
+  yield win.LoopUI.openPanel();
+
+  Assert.equal(win.document.getElementById("loop-notification-panel"), null, "Panel doesn't exist for private browsing");
+
+  win.close();
+});
```

Comment:
@Mardak , could you please review this tests? For some reason I'm getting an error when running them:

Error: Found an unexpected browser window at the end of test run